.github/workflows/release-python-bindings.yml

```diff
@@ -7,19 +7,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        # using macos-11 because of rocksdb
```

Comment:
Is that still an issue with rocksdb 20?

Reply:
Seems so https://github.com/iotaledger/wallet.rs/actions/runs/4192114912/jobs/7267286778